Marlin/language_da.h

```diff
@@ -190,56 +190,56 @@
 
 
 #if LCD_WIDTH > 19
-#define MSG_INFO_PRINT_COUNT              "Ant. Prints"
-#define MSG_INFO_COMPLETED_PRINTS         "Færdige"
-#define MSG_INFO_PRINT_TIME               "Total print tid"
-#define MSG_INFO_PRINT_LONGEST            "Længste print"
-#define MSG_INFO_PRINT_FILAMENT           "Total Extruderet"
+ #define MSG_INFO_PRINT_COUNT              "Ant. Prints"
+ #define MSG_INFO_COMPLETED_PRINTS         "Færdige"
+ #define MSG_INFO_PRINT_TIME               "Total print tid"
+ #define MSG_INFO_PRINT_LONGEST            "Længste print"
+ #define MSG_INFO_PRINT_FILAMENT           "Total Extruderet"
```

Comment:
This first group seems to be missing these:
```
MSG_INFO_MIN_TEMP
MSG_INFO_MAX_TEMP
MSG_INFO_PSU
MSG_DRIVE_STRENGTH
MSG_DAC_PERCENT
MSG_DAC_EEPROM_WRITE
MSG_FILAMENT_CHANGE_HEADER
MSG_FILAMENT_CHANGE_OPTION_HEADER
MSG_FILAMENT_CHANGE_OPTION_EXTRUDE
MSG_FILAMENT_CHANGE_OPTION_RESUME
```
Note that for items where the string length doesn't differ, they don't need to be inside the conditional and can be declared just once.